examples/remote_caching.py

```diff
@@ -8,10 +8,9 @@
 pg = ls.postgres.connect_env()
 
 name = "batting"
-path = ls.config.options.pins.get_path(name)
 
 right = (
-    ddb.read_parquet(path, table_name=name)
+    ls.examples.batting.fetch(backend=ddb)
```

Comment:
can we use `get_table_from_parquet` in situations like this?
```
ls.examples.core.get_table_from_parquet(name, backend=ddb, table_name=name)
```

Reply:
Yes, we could use it, but the user can also do:
```python
ls.examples.batting.fetch(backend=ddb, table_name=name)
```